libc/src/pthread/pthread_condattr_init.cpp

```diff
@@ -11,8 +11,8 @@
 #include "src/__support/common.h"
 #include "src/__support/macros/config.h"
 
+#include "hdr/time_macros.h"
```

Comment:
If the comments removed are still precise I kind of like keeping that style to tell at a glace _why_ we're including something, even if they're prone to becoming stale, and needlessly obfuscate `git blame`.  Here and below.  (just for time_macros.h; the hdr/types/*.h are self evident).